src/useAsyncStorage/index.js

```diff
@@ -0,0 +1,21 @@
+import AsyncStorage from '@react-native-community/async-storage';
+export const SEEN_PHRASE_KEY = '@seenPhrases';
+
+// Setting data to storage
+export const setSeenPhrasesToStorage = async (key, phrases) => {
```

Comment:
these names should be universal 
`setSeenPhrasesToStorage
getSeenPhraseFromStorage`

like 


`
setItemToStorage
getItemFromStorage`